components/VideoEmbedPreview/VideoEmbedPreview.tsx

```diff
@@ -1,7 +1,7 @@
 import getYouTubeID from "get-youtube-id";
 import { PreviewLayoutKey, PreviewProps } from "sanity";
 
-interface VideoEmbedPreviewProps extends PreviewProps<PreviewLayoutKey> {
+export interface VideoEmbedPreviewProps extends PreviewProps<PreviewLayoutKey> {
```

Comment:
TypeScript will probably complain, to solve it we can simply make `PreviewProps<PreviewLayoutKey>` as a [Partial](https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype) type

Reply:
Yea, thank you!

Reply:
Yea, thank you! @TonyMckes